packages/nx-plugin/src/schematics/builder/schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "id": "NxPluginBuilder",
+  "title": "Create a Builder for an Nx Plugin",
+  "type": "object",
+  "examples": [
+    {
+      "command": "g builder my-builder --project=my-plugin",
+      "description": "Generate libs/my-plugin/src/builders/my-builder"
+    }
+  ],
+  "properties": {
```

Comment:
Can you add support for `unitTestRunner`? This should support `jest` and `none`.

(And I just realized that the base schematic doesn't filter out the .spec files 🤦‍♂, that's my bad)

Reply:
Yeah sure, I will try and get that updated tonight.

Reply:
Do you think this change should be added to my other PR's as well?